lib/beaker-answers/versions/version30.rb

```diff
@@ -213,21 +205,19 @@ def generate_answers
       dashboard   = masterless ? nil : only_host_with_role(@hosts, 'dashboard')
       master      = masterless ? nil : only_host_with_role(@hosts, 'master')
       @hosts.each do |h|
-        if @options[:type] == :upgrade and h[:pe_ver] =~ /\A3.0/
-          # 3.0.x to 3.0.x should require no answers
-          the_answers[h.name] = {
-            :q_install => answer_for(@options, :q_install, 'y'),
-            :q_install_vendor_packages => answer_for(@options, :q_install_vendor_packages, 'y'),
-          }
-        else
-          the_answers[h.name] = host_answers(h, master, database, dashboard, @options)
-        end
-        if the_answers[h.name] && h[:custom_answers]
-          the_answers[h.name] = the_answers[h.name].merge(h[:custom_answers])
-        end
+        the_answers[h.name] =
+          if (@options[:type] == :upgrade) && h[:pe_ver] =~ /\A3.0/
+            {
+              :q_install => answer_for(@options, :q_install, 'y'),
+              :q_install_vendor_packages => answer_for(@options, :q_install_vendor_packages, 'y'),
+            }
+          else
+            host_answers(h, master, database, dashboard, @options)
+          end
+        the_answers[h.name] = the_answers[h.name].merge(h[:custom_answers]) if the_answers[h.name] && h[:custom_answers]
```

Comment:
Let's go back to an if/end block here rather than a terminal modifier.

Reply:
https://rubocop.readthedocs.io/en/latest/cops_style/#styleifunlessmodifier disabled